changelog/README.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
 ## 6th December 2024
+* Added new operation [Get all identity documents](../operations/identitydocuments.md#get-all-identity-documents).
+* Added new operation [Add identity documents](../operations/identitydocuments.md#add-identity-documents).
+* Added new operation [Update identity documents](../operations/identitydocuments.md#update-identity-documents).
+* Added new operation [Delete identity documents](../operations/identitydocuments.md#delete-identity-documents).
+* Added new operation [Clear identity documents](../operations/identitydocuments.md#clear-identity-documents).

```

Comment:
nitpick: Consider grouping this under a parent item, like "Added new operations to work with [Identity documents]" and then just list the operations without the "Added new operation" part.